Update hello_world_workflow.md (#25)
movestore/movestore.github.io: hello_world_workflow.md

# Hello world! - A Workflow example

Please follow the instructions below and create your first running Workflow on MoveApps. Once you have [registered](https://www.moveapps.org/register) in MoveApps, log in and you can start.
In the instructions below, buttons on the website are shown in `orange`.

## Download your data from Movebank
From the main menu in your `Dashboard`, select `Workflows` and then `Create New Workflow`. After providing a name for the Workflow, you will be asked to `Select A Data Source`. Choose `Movebank Location` and provide your [Movebank](https://www.movebank.org) login details.

![](files/HelloWorld_selectDataSource.png ':size=80%')

> In a succession of settings windows you can select a study from which you have access to download data, individual animals, a time window of required data and an option how to deal with duplicate locations.

Reviewer: ```suggestion
In a succession of settings windows you can select a study from which you have access to download data, individual animals, and an option to set a time window of required data.
```

Reviewer: The option of how do deal with duplicated locations is a relict of a older version. We will remove this option in the next release. Of course you can't know this :)

![](files/HelloWorld_selectStudy.png)

For shorter run times in this example let us select data of only two animals, say 700 and 712. As the data sets are not very dense, for now a time range selection is not necessary. Select `Finish` in the final Overview window. Your first Workflow, made up of one App, is now available for running.

If you like, go ahead and click on `Start Workflow` on the top right and, after the run is finished, explore the downloaded data (Output Details) in the `Cargo Agent`, which can be accessed via the small green circle appearing on the right side of the App Container (red arrow) once the run is finished. Explore the coloured dots (by hovering over them) that indicate the status of each App's performance (blue arrow). On the right side of the Workflow name there is also a action point which will indicate if the workflow instance is "running" or "stopped" (yellow arrow). Note that if the workflow contains a Shiny App its status will always be "running", also after finishing, in order to allow the interaction with the App UI.

![](files/HelloWorld_MoveApps_CargoAgent.png ':size=80%')

## Plot your positions in a Shiny User Interface

> Now you add an app to visualise the downloaded data. Let us stay simple and use the visualisation App `Static Map`. From the main menu in your `Dashboard`, select `App Browser` and search for the App. This App requires the input data type `moveStack`. 

Reviewer: ```suggestion
Now you add an App to visualise the downloaded data. Let us stay simple and use the visualisation App `Static Map`. From the main menu in your `Dashboard`, select `App Browser` and search for the App. This App requires the input data type `moveStack`. 
```
